samples/web/content/apprtc/js/call.js

```diff
@@ -180,20 +180,19 @@ Call.prototype.connectToRoom_ = function(mediaPromise, turnPromise) {
   // already registered with GAE.
   Promise.all([channelPromise, registerPromise]).then(function() {
     this.channel_.register(this.params_.roomId, this.params_.clientId);
+    // We only create a peer connection once we have media and TURN. Since right
```

Comment:
Done.

On Wed, Jan 14, 2015 at 1:17 PM, Justin Uberti <notifications@github.com>
wrote:

> In samples/web/content/apprtc/js/call.js
> <https://github.com/GoogleChrome/webrtc/pull/377#discussion-diff-22970670>
> :
>
> > @@ -180,20 +180,19 @@ Call.prototype.connectToRoom_ = function(mediaPromise, turnPromise) {
> >    // already registered with GAE.
> >    Promise.all([channelPromise, registerPromise]).then(function() {
> >      this.channel_.register(this.params_.roomId, this.params_.clientId);
> > +    // We only create a peer connection once we have media and TURN. Since right
>
> Update this comment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/webrtc/pull/377/files#r22970670>.
>



-- 
Jiayang Liu